doc/12.-Active-Directory-Watch-Daemon.md

```diff
@@ -0,0 +1,119 @@
+# Active Directory Watch Daemon
```

Comment:
I would move the Command line reference at the end, and so, this one is 11, the reference is 12.